content/docs/gke/deploy/monitor-iap-setup.md

```diff
@@ -182,18 +115,6 @@ problems:
     * The `istio-ingressgateway-XX` pods should be running
     * Check the logs of `backend-updater-0`, `ingress-bootstrap-XX`, `iap-enabler-XX` to see if there is any error
 
-1. Now that the certificate exists, the Ingress resource should report that it 
-  is serving on HTTPS:
-
-    ```
-    kubectl -n istio-system get ingress
-    NAME            HOSTS                                                        ADDRESS          PORTS     AGE
-    envoy-ingress   mykubeflow.endpoints.myproject.cloud.goog   35.244.132.159   80, 443   1d
-    ```
-
-    If you don't see port 443, look at the Ingress events using 
-    `kubectl describe` to see if there are any errors.
-
 
 1. Try accessing Cloud IAP at the fully qualified domain name in your web 
```

Comment:
Lets add a NextSteps section like we have here
https://www.kubeflow.org/docs/started/getting-started/#next-steps

Here are some links to include

* The Troubleshooting Guide for Kubeflow https://www.kubeflow.org/docs/gke/troubleshooting-gke/

* Docs to share cluster access
  https://www.kubeflow.org/docs/other-guides/multi-user-overview/#onboard-new-user

* Cloud IAP Docs https://cloud.google.com/iap/docs/

Reply:
done